deploy/lighthouse/cluster_role.yaml

```diff
@@ -17,7 +17,7 @@ rules:
 - apiGroups:
   - lighthouse.submariner.io
   resources:
-  - multiclusterservices
+  - "*"
```

Comment:
Should change this in broker/rbac.go also. It will be useful when we add serviceImport and endpointslice as those will exist on broker too.

Reply:
ack.